### PR TITLE
ft/day 5

### PR DESCRIPTION
**Pull request by Samael19IT5**

### CODE REVIEW

**Comment by Samael19IT5** [comment]
Add bootstrap & deploy to heroku

File: Gemfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ gem "rails", "~> 7.0.3", ">= 7.0.3.1"
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem "sprockets-rails"
 
-# Use sqlite3 as the database for Active Record
-gem "sqlite3", "~> 1.4"
+# Use pg as the database for Active Record
+gem "pg", "~> 1.1"
```
Comment (ttduongtran) [inline Gemfile:13]: Great!

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -44,6 +44,6 @@ def destroy
 
   private
   def post_params
-    params.require(:post).permit(:title, :description, :kind, :author)
+    params.require(:post).permit(:category_id, :title, :description, :author)
```
Comment (ttduongtran) [inline app/controllers/posts_controller.rb:47]: Nice

File: app/views/categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <% @categories.each do |category| %>
+    <tr>
+      <th scope="row">
+        <%= index %></th>
+      <td><%= category.title %></td>
+      <td><%= category.description %></td>
+      <td><%= category.category_type %></td>
+      <td>
+        <%= link_to "Show", category, class: 'btn btn-primary mx-2' %>
+        <%= link_to "Edit", edit_category_path(category), class: 'btn btn-success mx-2' %>
```
Comment (ttduongtran) [inline app/views/categories/index.html.erb:23]: Seems to understand how to use bootstrap classes. Good work

**Review by ttduongtran — APPROVED** [review]
Nice UI

**Pull request closed without merging**